docs/reference/overview_of_strategies.rst

```diff
@@ -250,7 +250,91 @@ This strategy plays randomly (disregarding the history of play).
 Axelrod's second tournament
 ---------------------------
 
-Work in progress.
+The code for Axelrod's second touranment was originally published by the
+`University of Michigan Center for the Study of Complex Systems <http://lsa.umich.edu/cscs/>`_
+and is now available from
+`Robert Axelrod's personal website <http://www-personal.umich.edu/~axe/research/Software/CC/CC2.html>`_.
+
+The Axelrod-Python organisation has published a
+`modified version of the original code <https://github.com/Axelrod-Python/TourExec>`_.
+In the following table, links to original code point to the Axelrod-Python
+repository.
+
+.. include:: fortan_keys.rst
+
+.. csv-table:: Strategies in Axelrod's second tournament
+   :header: "Original Code", "Name", "Axelrod Library Name"
+   :widths: 27, 24, 60
+
+   "GRASR_", "Grasr", "Not Implemented"
+   "K31R_", "Grisell", "Not Implemented"
+   "K32R_", "Kluepfel", "Not Implemented"
+   "K33R_", "Rabbie", "Not Implemented"
+   "K34R_", "Friedman", "Not Implemented"
+   "K35R_", "Getzler", "Not Implemented"
+   "K36R_", "Hotz", "Not Implemented"
+   "K37R_", "Lefevre", "Not Implemented"
+   "K38R_", "Weiderman", "Not Implemented"
+   "K39R_", "Almy", "Not Implemented"
+   "K40R_", "Adams", "Not Implemented"
+   "K41R_", "Weiner", "Not Implemented"
+   "K42R_", "Borufsen", "Not Implemented"
+   "K43R_", "Anderson", "Not Implemented"
+   "K44R_", "Adams", "Not Implemented"
+   "K45R_", "Mcgurrin", "Not Implemented"
+   "K46R_", "Eatherley", ":class:`axelrod.strategies.axelrod_second.Eatherley`"
+   "K47R_", "Hufford", "Not Implemented"
+   "K48R_", "Hufford", "Not Implemented"
+   "K49R_", "Cave", "Not Implemented"
+   "K50R_", "Rik", "Not Implemented"
+   "K51R_", "Colbert", "Not Implemented"
+   "K52R_", "Smith", "Not Implemented"
+   "K53R_", "Nussbacher", "Not Implemented"
+   "K54R_", "Robertson", "Not Implemented"
+   "K55R_", "Newman", "Not Implemented"
+   "K56R_", "Quayle", "Not Implemented"
+   "K57R_", "Nydegger", ":class:`axelrod.strategies.axelrod_first.Nydegger`"
+   "K58R_", "Rowsam", "Not Implemented"
+   "K59R_", "Downing", "Not Implemented"
+   "K60R_", "Graaskamp", "Not Implemented"
+   "K61R_", "Champion", ":class:`axelrod.strategies.axelrod_second.Champion`"
+   "K62R_", "Hollander", "Not Implemented"
+   "K63R_", "Duisman", "Not Implemented"
+   "K64R_", "Yamachi", "Not Implemented"
+   "K65R_", "Batell", "Not Implemented"
+   "K66R_", "Mikkelson", "Not Implemented"
+   "K67R_", "Feathers", "Not Implemented"
+   "K68R_", "Leyvraz", "Not Implemented"
+   "K69R_", "Joss", ":class:`axelrod.strategies.axelrod_first.Joss`"
+   "K70R_", "Pebly", "Not Implemented"
+   "K71R_", "Hill", "Not Implemented"
+   "K72R_", "White", "Not Implemented"
+   "K73R_", "Zimmerman", "Not Implemented"
+   "K74R_", "Friedland", "Not Implemented"
```

Comment:
I'm guessing that's why the second of those has 'XX' in its code.

I suggest we leave it as is for the moment. Come the point when we translate this into Python, we'll have to come up with some sort of unique name for them but, until then, it's unimportant.